draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -258,24 +252,10 @@ to deploy, but deployment would not be recommended because it
 could lead to unnecessary delays for the connections attempting to use
 Quick-Start. The Quick-Start method is discussed as an example in {{?RFC9049}}.
 
-For authors of alternate congestion control schemes who are not ready
-to bring their congestion control mechanisms to the IETF for
-standardization (either as Experimental or as Proposed Standard), one
-possibility would be to submit an internet-draft that documents the
-alternate congestion control mechanism for the benefit of the IETF
-and IRTF communities.  This is particularly encouraged in order to
-ensure algorithm specifications are widely disseminated to facilitate
-further research.  Such an internet-draft could also be
-considered for publication as an Informational RFC, as a first step in the process
-towards standardization.  Such a document would be expected to
-carry an explicit warning against using the scheme in the global
-Internet.
-
-Note: we are not changing the RFC publication process for non-IETF
-produced documents (e.g., those from the IRTF or Independent
-Submissions via the RFC-Editor).  However, we would hope the
-guidelines in this document inform the IESG as they consider whether
-to add a note to such documents.
+Though out of scope of this document, congestion control proponents may also
+seek publication of an Informational or Experimental RFC via the Internet
+Research Task Force (IRTF). In general, these proposals are expected to be less
```

Comment:
And via the ISE?
... In the past, we have had a number of requests to publish via the ISE, for documentation, because they did not wish to allow the IETF change-control of their method. That's maybe NOT one to highlight though?